stable/dask-distributed/values.yaml

```diff
@@ -0,0 +1,42 @@
+# Default values for dask.
+# This is a YAML-formatted file.
+# Declare name/value pairs to be passed into your templates.
+# name: value
+
+Scheduler:
+  Name: scheduler
+  Image: "daskdev/dask"
+  ImageTag: "latest"
+  Replicas: 1
+  Component: "dask-scheduler"
+  Cpu: "500m"
+  Memory: "1024Mi"
+  ServicePort: 8786
+  ContainerPort: 8786
+
+WebUI:
+  Name: webui
+  ServicePort: 8787
+  ContainerPort: 8787
+
+Worker:
+  Name: worker
+  Image: "daskdev/dask"
+  ImageTag: "latest"
+  Replicas: 3
+  Component: "dask-worker"
+  Cpu: "500m"
+  Memory: "1024Mi"
```

Comment:
Please excuse my ignorance here, but what does 500m mean, milli-cpus?  Also, do we want more ram than this by default?  I'm inclined to go with slightly larger workers by default to avoid memory issues.  Users very typically move around chunks of data that are a few hundred megs.